src/app/button-group/button-group.demo.module.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {NgModule} from "@angular/core";
+import {CommonModule} from "@angular/common";
+import {ClarityModule} from "../../clarity-angular";
+import {ROUTING} from "./buttons.demo.routing";
+import {BasicButtonGroupDemo} from "./angular/basic-structure/basic-button-group";
+import {HideShowOverflowToggleDemo} from "./angular/hide-show-overflow-toggle/hide-show-overflow-toggle";
+import {IconButtonGroupDemo} from "./angular/icon-buttons/icon-button-group";
+import {MenuDirectionsDemo} from "./angular/menu-directions/menu-directions";
+import {MixedButtonGroupDemo} from "./angular/mixed-buttons/mixed-button-group";
+import {MoveAllInMenuDemo} from "./angular/move-all-in-menu/move-all-in-menu";
+import {MoveButtonInMenuDemo} from "./angular/move-button-in-menu/move-button-in-menu";
+import {MoveMultipleButtonInMenuDemo} from "./angular/move-multiple-buttons-in-menu/move-multiple-button-in-menu";
+import {ProjectionUpdateTest1Demo} from "./angular/projection-update-test-1/projection-update-test-1";
+import {ProjectionUpdateTest6Demo} from "./angular/projection-update-test-6/projection-update-test-6";
+import {ProjectionUpdateTest5Demo} from "./angular/projection-update-test-5/projection-update-test-5";
+import {ProjectionUpdateTest4Demo} from "./angular/projection-update-test-4/projection-update-test-4";
+import {ProjectionUpdateTest3Demo} from "./angular/projection-update-test-3/projection-update-test-3";
+import {ProjectionUpdateTest2Demo} from "./angular/projection-update-test-2/projection-update-test-2";
+import {StaticButtonGroupBasicStructureDemo} from "./static/basic-structure/basic-structure";
+
+import {ButtonGroupCheckboxesDemo} from "./static/checkbox/button-group-checkboxes";
+import {ButtonGroupRadiosDemo} from "./static/radio/button-group-radios";
+import {ButtonGroupIconsDemo} from "./static/icons/button-group-icons";
+import {ButtonGroupIconsTextDemo} from "./static/icons-with-text/button-group-icon-text";
+import {ButtonGroupTypes} from "./static/types/button-group-types";
+
+import {ButtonGroupStaticDemo} from "./static/button-group-static";
+import {ButtonGroupAngularDemo} from "./angular/button-group-angular";
+import {ButtonGroupDemo} from "./button-group.demo";
+import {StaticMenuDirectionsDemo} from "./static/menu-directions/menu-directions";
+
+@NgModule({
+    imports: [
+        CommonModule,
+        ClarityModule,
```

Comment:
I think this is just on the website branch. I dont see any other demo module having this.